activiti-engine/src/test/java/org/activiti/engine/test/bpmn/event/error/BoundaryErrorEventTest.java

```diff
@@ -54,14 +54,26 @@ public void testThrowErrorWithEmptyErrorCode() {
     }
   }
 
-  @Deployment
+  public void testThrowErrorWithoutErrorCode() {
+    try {
+      repositoryService.createDeployment().addClasspathResource("org/activiti/engine/test/bpmn/event/error/BoundaryErrorEventTest.testThrowErrorWithoutErrorCode.bpmn20.xml").deploy();
+      fail("ActivitiException expected");
+    } catch (ActivitiException re) {
+    }
+  }
+
   public void testCatchErrorOnEmbeddedSubprocessWithEmptyErrorCode() {
+    try {
     testCatchErrorOnEmbeddedSubprocess();
+    } catch (ActivitiException re) {
```

Comment:
Is this test suppose the throw exception or not? Note that you are not calling `fail` inside the `try` block, so it's going to succeed in both ways.